charts/cryostat/templates/alpha_config.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-alpha-config
+data:
+  alpha_config.yaml: |-
+    server:
+      BindAddress: http://0.0.0.0:4180
+    upstreamConfig:
+      proxyRawPath: true
+      upstreams:
+        - id: cryostat
+          path: /
+          uri: http://cryostat:8181
```

Comment:
I think the hostnames here (`cryostat`, `grafana`, `s3`) in the URLs should actually just be `localhost`. They are distinct hostnames in the smoketest version because there each of these containers are run separately and are addressable by their container names. In this case we are deploying them all as containers within one Pod, so I think just using `localhost` is what will work most simply.